### PR TITLE
fix demo doc

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
model does not work on HDDL/MyriadX, so do not mention these devices in demo documentation

### CODE REVIEW

File: demos/python_demos/text_spotting_demo/README.md [hunk 1]
```diff
@@ -68,10 +68,10 @@ Options:
...
   -i "<path>"           Required. Input to process.
   -d "<device>", --device "<device>"
-                        Optional. Specify the target device to infer on: CPU,
-                        GPU, FPGA, HDDL or MYRIAD. The demo will look for a
-                        suitable plugin for device specified (by default, it
-                        is CPU).
+                        Optional. Specify the target device to infer on, i.e. CPU, GPU.
+                        The demo will look for a suitable plugin for device specified
+                        (by default, it is CPU). Please refer to OpenVINO documentation
+                        for the list of devices supported by the model.
```
Comment (IRDonch) [inline demos/python_demos/text_spotting_demo/README.md:74]: You need to change the help text in the code accordingly.

Comment (vladimir-dudnik) [reply demos/python_demos/text_spotting_demo/README.md:74]: yes, fixed as well

Comment (IRDonch) [reply demos/python_demos/text_spotting_demo/README.md:74]: You didn't change it in the same way. The text needs to be identical (in fact, you're supposed to just run the demo with `--help` and paste the output into the README).

Comment (vladimir-dudnik) [reply demos/python_demos/text_spotting_demo/README.md:74]: that's right, corrected

File: demos/python_demos/text_spotting_demo/text_spotting_demo.py [hunk 2]
```diff
@@ -82,9 +82,10 @@ def build_argparser():
                       help='Required. Input to process.',
                       required=True, type=str, metavar='"<path>"')
     args.add_argument('-d', '--device',
-                      help='Optional. Specify the target device to infer on: CPU, GPU, FPGA, HDDL or MYRIAD. '
+                      help='Optional. Specify the target device to infer on, i.e : CPU, GPU. '
                            'The demo will look for a suitable plugin for device specified '
-                           '(by default, it is CPU).',
+                           '(by default, it is CPU). Please refer to OpenVINO documentation '
+                           'for the list of devices supported be the model.',
```
Comment (IRDonch) [inline demos/python_demos/text_spotting_demo/text_spotting_demo.py:88]:
```suggestion
                           'for the list of devices supported by the model.',
```

**Pull request merged**